lib/metrics/eventLoopLag.js

```diff
@@ -16,7 +17,8 @@ module.exports = registry => {
 	const gauge = new Gauge({
 		name: NODEJS_EVENTLOOP_LAG,
 		help: 'Lag of event loop in seconds.',
-		registers: registry ? [registry] : undefined
+		registers: registry ? [registry] : undefined,
+		aggregator: AGGREGATORS.AVERAGE
```

Comment:
I suppose there are some special-cases (like `nodejs_version_info`), but can a sensible default be set?